<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsLS</class>
 <widget class="QWidget" name="QtWidgetsLS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>292</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>292</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询地址输入</string>
  </property>
  <widget class="QLabel" name="labelHead">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>根据地址查询乘车线路</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>起点地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>终点地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>230</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxEnd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsLS</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxStart</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>QtWidgetsLS</receiver>
   <slot>startsearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxEnd</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>QtWidgetsLS</receiver>
   <slot>endsearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSearch</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsLS</receiver>
   <slot>inputloc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputloc()</slot>
  <slot>startsearch()</slot>
  <slot>endsearch()</slot>
 </slots>
</ui>
